mantis/analysis/register.py

```diff
@@ -267,7 +267,7 @@ def find_lir(registered_zyx: np.ndarray, plot: bool = False) -> Tuple:
     # Find the lir Z
     zyx_shape = registered_zyx.shape
     registered_zx_bool = registered_zyx.transpose((2, 0, 1)) > 0
-    registered_zx_bool = registered_zx_bool[zyx_shape[0] // 2].copy()
+    registered_zx_bool = registered_zx_bool[zyx_shape[2] // 2].copy()
```

Comment:
Can you explain what is happening here? `registered_zyx.transpose((2, 0, 1))` should result in a 3D array rather than a 2D array as the `zx_bool` label suggests. Could you index `zyx_shape` from the back, e.g. `zyx_shape[-2]` such that it's clear if you're taking the X_size or Z_size?

Reply:
changed. This is to estimate the Z- LIR since we do rotate the volume in Z through the optimizer. Here, we take the middle of the ZX projection across X as sample to estimate LIR in Z.